test/unit/annotations/metaExtension.spec.ts

```diff
@@ -19,15 +19,15 @@ test("MetaExtension", () => {
         }
     `;
 
-    const result = util.transpileAndExecute(
-        'return debug.getregistry()["_LOADED"].test();',
-        undefined,
-        undefined,
-        tsHeader,
-        true
-    );
-
-    expect(result).toBe(5);
+    // Can't use expectToMatchJsResult because above is not valid TS/JS
+    util.testModule`
+       export default debug.getregistry()["_LOADED"].test();
+    `
+        .setTsHeader(tsHeader)
+        .ignoreDiagnostics([annotationDeprecated.code])
+        .setReturnExport("default")
+        .expectToEqual(5)
+        .getLuaExecutionResult();
```

Comment:
I think you should be able to remove `getLuaExecutionResult()` here, `expectToEqual` should call it internally.